configuration/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+# Create a library called "Hello" which includes the source file "hello.cxx".
+# The extension is already found. Any number of sources could be listed here.
+add_library (ConfigHandler ConfigHandler.cpp)
+set(CMAKE_CXX_FLAGS "-lyaml-cpp")
```

Comment:
This is compiler dependant, please use `target_link_libraries`

Reply:
Do I use it like this ?
`target_link_libraries(yaml-cpp)`

Reply:
'cause I don't quite understand the cmake doc about this